js/popup.js

```diff
@@ -51,17 +58,23 @@
 					operand2 = parseInt(operand2);
 				}
 				if(result[i] == '/') {
-					stack.push(operand2/operand1);
+					ele = operand2/operand1;
+					//stack.push(operand2/operand1);
 				} else if(result[i] == '*') {
-					stack.push(operand2*operand1);
+					ele = operand2*operand1;
+					//stack.push(operand2*operand1);
 				} else if(result[i] == '+') {
-					stack.push(operand2+operand1);
+					ele = operand2+operand1;
+					//stack.push(operand2+operand1);
 				} else if(result[i] == '-') {
-					stack.push(operand2-operand1);
+					ele = operand2-operand1;
+					//stack.push(operand2-operand1);
 				}
+				
+				stack.push(ele);
 			}
+			console.log(ele);
 		}
-		// Not working for case 5+8/4+5*2-8/3+2 try solving it @Ekta!!
 		$('#ans').text(stack.pop());
 	})
-}).call(this);
```

Comment:
there is a new line at the end of this line remove this and send me a pull request again